README.md

```diff
@@ -48,6 +57,23 @@ pub trait Instance {
 }
 ```
 
+The `container::Engine` trait provides a simplified API:
+
+```rust
+pub trait Engine: Clone + Send + Sync + 'static {
+    /// The name to use for this engine
+    fn name() -> &'static str;
+    /// Run a WebAssembly container
+    fn run(&self, ctx: impl RuntimeContext, stdio: Stdio) -> Result<i32>;
```

Comment:
Just an opinion: I like to rename this function to `run_wasi(...)` to better describe its use cases.

Reply:
Done